backend/tcs-integration/Controllers/SessionsController.test.cs

```diff
@@ -0,0 +1,165 @@
+        {
+            _factory = factory;
+            fixture = new Fixture().Customize(new AutoMoqCustomization());
+        }
+
+        private async Task<UserDto> Login(HttpClient client)
+        {
+            var logInResponse = await client.PostAsJsonAsync("api/users/authenticate", new { username = "tcs", password = "Develop@90" });
+            Assert.Equal(HttpStatusCode.OK, logInResponse.StatusCode);
+            var user = await logInResponse.Content.ReadAsAsync<UserDto>();
+            return user;
+        }
+
+        [Fact]
+        public async void GET_sessions_ShouldReturnListOfSessions()
+        {
+            var client = _factory.CreateClient();
+            var user = await Login(client);
+            var request = new HttpRequestMessage(HttpMethod.Get, "api/sessions");
+            request.Headers.Add("Authorization", $"Bearer {user.Token}");
+
+            var response = await client.SendAsync(request);
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+            var sessions = await response.Content.ReadAsAsync<Paging<Session>>();
+
+            Assert.NotNull(sessions);
+        }
+
+
+        [Fact]
+        public async void GET_sessions_id_ShouldReturnSessionWithMatchingId()
+        {
+            var sessionId = 1;
+            var client = _factory.CreateClient();
+            var user = await Login(client);
+            var request = new HttpRequestMessage(HttpMethod.Get, $"api/sessions/{sessionId}");
+            request.Headers.Add("Authorization", $"Bearer {user.Token}");
+
+            var response = await client.SendAsync(request);
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+            var session = await response.Content.ReadAsAsync<Session>();
+
+            Assert.Equal(sessionId, session.ID);
+        }
+        
+        [Fact]
+        public async void GET_outtime_null_ShouldReturnListOfSessionsWhereOutTimeIsNull()
+        {
+            var client = _factory.CreateClient();
+            var user = await Login(client);
+            var request = new HttpRequestMessage(HttpMethod.Get, "api/sessions/signedin");
+            request.Headers.Add("Authorization", $"Bearer {user.Token}");
+
+            var response = await client.SendAsync(request);
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+            var sessions = await response.Content.ReadAsAsync<IEnumerable<Session>>();
+
+            Assert.All(sessions, s => Assert.Null( s.OutTime ));
+        }
+
+        [Fact]
+        public async void POST_sessions_ShouldReturn201WithCreatedSession()
+        {
+            var client = _factory.CreateClient();
+            var session = new SessionCreateDTO()
+            {
+                InTime = DateTime.Now,
+                PersonId = 1,
+                Tutoring = true,
+                SelectedClasses = new List<Class>() { new Class { CRN = 4, DepartmentCode = 2, Name = "ENGLISH LIT", ShortName = "English 101" } } ,
```

Comment:
This should be a list of the Id.
EX: SelectedClasses = [123146, 46798, 231645]
This will minimize what it takes to construct the SessionCreateDTO on the frontend.

Reply:
Both SelectedClasses and Reasons should behave in this way.